### PR TITLE
Add Launchpad terms of service

### PR DESCRIPTION
**Pull request by cjwatson**
This is lifted directly from https://help.launchpad.net/Legal, except for the privacy policy section; that will become a Launchpad-specific privacy notice under /legal/dataprivacy once legal have finished
drafting it.

The text has been through a [round of review](https://docs.google.com/document/d/1b_MkB4ejhOQ1IWtXIOBDa38G8pw2NhJFF_TfTlnzSqE/edit) by legal, and they've requested that it be moved to www.ubuntu.com.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the relevant part of the site locally in your web browser at: [http://0.0.0.0:8001/legal/terms-and-policies](http://0.0.0.0:8001/legal/terms-and-policies) and confirm that the Launchpad terms section looks reasonable

### CODE REVIEW

File: templates/legal/terms-and-policies/launchpad-terms-of-service.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+<div class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1 id="title">Launchpad terms of service</h1>
+
+      <section>
+        <h2 id="trademark-and-logo">Trademark and logo licence</h2>
+        <p>"<em>Launchpad</em>" and the Launchpad "Gem" logo are trademarks of Canonical Ltd.</p>
+        <p>The Launchpad "Gem" logo was created by <a href="https://launchpad.net/~etretyak">Eugene Tretyak</a> as part of the <a href="https://help.launchpad.net/logo">Launchpad Logo Contest</a>. The copyright in the Launchpad logo is owned by Canonical Ltd. and is licensed under a <a href="http://creativecommons.org/licenses/by-nd/2.0/uk/">Creative Commons Attribution-No Derivative Works 2.0 UK: England & Wales License</a>.</p>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/launchpad-terms-of-service.html]: @cjwatson all external links need to be decorated with a `class="p-link--external"`

File: templates/legal/terms-and-policies/launchpad-terms-of-service.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        <p>You can use one of the images from the <a href="https://help.launchpad.net/BadgeKit">Launchpad Badge Kit</a> to show that your project uses Launchpad. Canonical may revoke your permission to use these images if you make modifications to the images or use them incorrectly (e.g., on a project that does not use Launchpad.net).</p>
+      </section>
+
+      <section>
+        <h2 id="help-dev-and-news">Help, Dev, and News Content Licence</h2>
+        <p>The documentation contained in the <a href="https://help.launchpad.net/">Launchpad Help wiki</a>, the <a href="https://dev.launchpad.net/">Launchpad Development wiki</a>, and on the <a href="http://blog.launchpad.net/">Launchpad blog</a> is owned by Canonical Ltd. and is licensed under a <a href="http://creativecommons.org/licenses/by/2.0/uk/">Creative Commons Attribution 2.0 UK: England & Wales License</a>.</p>
```
Comment (pmahnke) [inline templates/legal/terms-and-policies/launchpad-terms-of-service.html]: @cjwatson the '&' needs to be encoded `&amp;`

**Comment by pmahnke** [comment]
@frankiegranato you or I need to create and update a few copy docs based on this!

**Comment by cjwatson** [comment]
@pmahnke Thanks for the review; I've made the requested changes.

**Pull request merged**